internal/connector/internal/workers/db_mgr.go

```diff
@@ -0,0 +1,29 @@
+package workers
+
+import (
+	"context"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/db"
+	serviceError "github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/errors"
+)
+
+func InDBTransaction(ctx context.Context, f func(ctx context.Context) error) (rerr *serviceError.ServiceError) {
```

Comment:
Shared between `connector`'s and `processor`'s so moved out of the `connectors_mgr.go` file.